uniffi_bindgen/src/bindings/swift/gen_swift/mod.rs

```diff
@@ -449,7 +456,7 @@ pub mod filters {
 
     pub fn type_name(codetype: &impl CodeType) -> Result<String, askama::Error> {
         let oracle = oracle();
-        Ok(codetype.type_label(oracle))
+        Ok(escape_reserved_word_identifier(codetype.type_label(oracle)))
```

Comment:
I think this is causing the tests to fail because it's quoting types that shouldn't be quoted like `String?` or `[Int32]`.  I think this should be moved to the `CodeType` impls themselves.  We should quote user-defined types like `Enum`, `Error`, `Record`, `Object`, `CallbackInterface` etc.  Primitives, compounds, shouldn't be quoted.